pkgs/stdenv/generic/setup.sh

```diff
@@ -116,13 +118,14 @@ exitHandler() {
 
     if [ -n "${showBuildStats:-}" ]; then
         times > "$NIX_BUILD_TOP/.times"
-        local -a times=($(cat "$NIX_BUILD_TOP/.times"))
+        local -a buildTimesArray
+        IFS=" " read -r -a buildTimesArray < "$NIX_BUILD_TOP/.times"
```

Comment:
FWIW I thought the original was OK, and this shouldn't be affected by OSH?  I would be interested to learn otherwise.

Reply:
This one was just a shell check